src/modules/app.js

```diff
@@ -0,0 +1,85 @@
+import { HomePage } from './components/homepage';
+import { Menu } from './components/menu';
+import { SpecialMenu } from './components/special_menu';
+import { Contact } from './components/contact';
+import { About } from './components/about';
+
+const content = document.createElement('div');
+
+const navbar = document.createElement('div');
+navbar.className = 'navbar navbar-expand-lg bg-dark fixed-top';
+
+function setAttributes(el, attrs) {
+  Object.keys(attrs).forEach(key => el.setAttribute(key, attrs[key]));
+}
+
+// brand
```

Comment:
It’s better to delete commented code. If you need to review a previous version of your code, use git history, your code is clear enough to understand 🙂 ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

**Kindly delete all comments in this file**